pkg/asset/imagebased/image/baseiso_test.go

```diff
@@ -0,0 +1,95 @@
+package image
+
+import (
+	"context"
+	"crypto/rand"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/coreos/stream-metadata-go/stream"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/openshift/installer/pkg/asset"
+)
+
+func TestBaseIso_Generate(t *testing.T) {
```

Comment:
Add test case that verifys Load works as expected

Reply:
As explained [here](https://github.com/openshift/installer/pull/8679#discussion_r1662738435) I'll remove `Load` as it's redundant (in case I end up actually needing `Load` I'll of course add the respective test as you suggested) :+1: